frontend/src/features/admin-form/common/mutations.ts

```diff
@@ -388,3 +389,32 @@ export const usePreviewFormMutations = (formId: string) => {
     submitStorageModeFormMutation,
   }
 }
+
+export const useFormFeedbackMutations = (formId: string, formTitle: string) => {
+  const toast = useToast({ status: 'success', isClosable: true })
+
+  const handleError = useCallback(
+    (error: Error) => {
+      toast.closeAll()
+      toast({
+        description: error.message,
+        status: 'danger',
+      })
+    },
+    [toast],
+  )
+
+  const downloadFormFeedbackMutation = useMutation(
+    () => downloadFormFeedback(formId ?? '', formTitle ?? ''),
```

Comment:
should we pass in the form Id and formTitle as parameters here instead of the hook?

May require adding type hints to `useMutation`. This allows the mutation function to be instantiated even when form is still loading.

```tsx
const downloadFormFeedbackMutation = useMutation(
    (formId, formTitle) => downloadFormFeedback(formId, formTitle),
    ...
)
```